torch/_subclasses/functional_tensor.py

```diff
@@ -97,6 +97,15 @@ class FunctionalTensor(torch.Tensor):
         torch.ops.aten._batch_norm_impl_index.default,  # type: ignore[has-type]
         torch.ops.aten.cudnn_batch_norm.default,  # type: ignore[has-type]
         torch.ops.aten.miopen_batch_norm.default,  # type: ignore[has-type]
+        torch.ops.aten.atleast_1d.default,  # type: ignore[has-type]
```

Comment:
how did you end up finding these?

Reply:
@pianpwk ran the schema check mode on pre-dispatch key stack